charts/fybrik/files/taxonomy/fybrik_application.json

```diff
@@ -14,6 +14,12 @@
                     "items": {
                         "type": "object",
                         "properties": {
+                            "flows": {
+                                "type": "array",
```

Comment:
Where do "flows" come from? It is not an array any longer, right?

Reply:
Fixed